src/elektra/redshift.ini

```diff
@@ -0,0 +1,234 @@
+; Elektra specification file for redshift@490ba2aae9cfee097a88b6e2be98aeb1ce990050 (= redshift v1.12 + some small commits)
+; This file is based on: https://github.com/ElektraInitiative/libelektra/blob/master/examples/spec/lcdproc/LCDd.ini
+;
+; A spec file "(...) defines how the configuration settings should be validated" (source: https://www.libelektra.org/tutorials/validate-configuration)
```

Comment:
(In this case) it also defines command-line arguments, defaults and describes the settings.

Reply:
I've incorporated these aspects into this comment.